lua/telescope/_extensions/file_browser/actions.lua

```diff
@@ -329,8 +329,12 @@ fb_actions.remove_file = function(prompt_bufnr)
   -- format printing adequately
   print "\n"
 
-  vim.ui.select({ "Yes", "No" }, { prompt = "Remove Selected Files" }, function(_, idx)
-    if idx == 1 then
+  vim.ui.input({ prompt = "Remove selected files (Y/N): " }, function(user_answer)
```

Comment:
Slight nit for conventions: can we rename `user_answer` to `input` to stay streamlined with `vim.ui.input` docs?

What about also providing`completion` with `y/N` and `default = y`. In case of the latter, users can then just confirm with `CR`.

Thoughts?

Reply:
> Slight nit for conventions: can we rename user_answer to input to stay streamlined with vim.ui.input docs?
Sure, that is not a problem.

> What about also providing completion with y/N and default = y. In the case of the latter, users can then just confirm with CR.

If you want to default as yes, then it should be `(Y/n)`.

However, I don't agree with having **Yes** as the default, if this was a `continue` or `copy` operation, then **Yes** as default is ok, but for `delete` operations we should probably default to **No** for safety to prevent deletions by accident.

All other plugins handle default as **No** for delete operations, we should follow the same in my opinion.